solutions/projectstaffing/jgraph/core/src/main/scala/com/microsoft/graphdataconnect/skillsfinder/service/UserService.scala

```diff
@@ -165,12 +184,17 @@ class UserService {
   }
 
   def isCurrentUserAnAdmin(httpHeaders: HttpHeaders): Boolean = {
-    if (httpHeaders.toSingleValueMap.containsKey("x-ms-client-principal")) {
-      val clientPrincipalToken = httpHeaders.toSingleValueMap.asScala("x-ms-client-principal")
-      isUserPartOfAdminsGroup(clientPrincipalToken)
+    if (anonymousUser) {
+      true
```

Comment:
Why does the anonymous user have to be an admin ?

Reply:
yes, it's not needed, I changed it to false, thanks for your observation